qa/suites/orch/cephadm/workunits/task/test_mgmt_gateway.yaml

```diff
@@ -63,15 +63,27 @@ tasks:
         # retrieve mgmt hostname and ip
         MGMT_GTW_HOST=$(ceph orch ps --daemon-type mgmt-gateway -f json | jq -e '.[]' | jq -r '.hostname')
         MGMT_GTW_IP=$(ceph orch host ls -f json | jq -r --arg MGMT_GTW_HOST "$MGMT_GTW_HOST" '.[] | select(.hostname==$MGMT_GTW_HOST) | .addr')
-        # check mgmt-gateway health
+        # check mgmt-gateway external server health
         curl -k -s https://${MGMT_GTW_IP}/health
-        curl -k -s https://${MGMT_GTW_IP}:29443/health
+        # check mgmt-gateway internal server access is restricted
+        if [ "$(curl -k -s -o /dev/null -w "%{http_code}" https://${MGMT_GTW_IP}:29443/health)" -ne 400 ]; then
+            echo "The internal management gateway is unexpectedly reachable. Access should be restricted."
+            exit 1
+        fi
+
+        # retrieve monitoring hostnames
+        PROMETEHUS_HOST=$(ceph orch ps --daemon-type prometheus -f json | jq -e '.[]' | jq -r '.hostname')
+        ALERTMGR_HOST=$(ceph orch ps --daemon-type alertmanager -f json | jq -e '.[]' | jq -r '.hostname')
+
+        # check that prometheus and alertmanager are enforcing mTLS
+        openssl s_client -connect ${PROMETEHUS_HOST}:9095 -servername ${PROMETEHUS_HOST} 2>&1 | grep "alert certificate required"
```

Comment:
```suggestion
        openssl s_client -connect ${PROMETHEUS_HOST}:9095 -servername ${PROMETEHUS_HOST} 2>&1 | grep "alert certificate required"
```

Reply:
@guits plz can you clarify what't the proposed change?